### DISCUSSION TITLE
Can i change block manager config from plugin ?

### DISCUSSION THREAD

**Discussion by justmumu** [GrapesJS/grapesjs#4470] [Q&A]
I want to ask that can i change block manager's configuration from plugin ? For example, i want to change appendTo property from plugin.

**Comment by justmumu** [comment]
I think i found a working solution. But i am not sure that is cleanest way.

editor.BlockManager.getConfig().appendTo = 'foo';

**Comment by ronaldohoch** [comment]
Can you give more informations about?

**Reply by justmumu** [reply]
I was trying to change blockManager configuration from a loaded plugin. 
So, I supposed that plugins only reach its own configuration.

But i realize i can change blockManager configuration doing something like "editor.BlockManager.getConfig().appendTo = 'foo';"
I'm still wondering if this is the correct way.

Thanks..

**Reply by ronaldohoch** [reply]
Nice!
We change lot of stuff in plugins this way...
editor.SomeModule...
